### ISSUE TITLE
pypi / python3 issue on Windows

### ISSUE THREAD

**Issue by haesleinhuepf** [pypa/packaging.python.org#786]
Hi all,

tldr: reporting about a potentially easy-to-fix issue related to Windows 10. Consider replacing `python3` by `python` in user guide to make it work on Windows/conda.

Related to the documentation on this page:
https://packaging.python.org/tutorials/packaging-projects/

It mentions that his line should output a lot of stuff to the command line:
```
python3 setup.py sdist bdist_wheel
```

Unfortunately, some Windows python installations (with mini-conda in my case) don't have the `python3` symlink and thus, nothing happens:
![image](https://user-images.githubusercontent.com/12660498/87915206-1a077400-ca72-11ea-8e01-6db41ff4cb60.png)

By executing only `python3` one can reveal that the issue is that Windows does this: 
![image](https://user-images.githubusercontent.com/12660498/87915291-428f6e00-ca72-11ea-9d8b-729f25f4bc5d.png)

It opens the Windows store 🙄 

Would it make sense to point Windows users to this potential issue? Alternatively, could you replace `python3` by just `python` because it works:
![image](https://user-images.githubusercontent.com/12660498/87915471-87b3a000-ca72-11ea-8147-e32ec799bf96.png)

Let me know if you need any further information.

Cheers,
Robert

**Comment by webknjaz** [comment]
@haesleinhuepf the docs now have some Windows command examples. Is this still an issue?

**Comment by pradyunsg** [comment]
I'm going to asssume that OP figured out what they needed to do here. :)
